subsys/bluetooth/controller/ll_sw/nordic/lll/lll_df.c

```diff
@@ -307,17 +290,17 @@ static int init_reset(void)
 
 #if defined(CONFIG_BT_CTLR_DF_ADV_CTE_TX) || defined(CONFIG_BT_CTLR_DF_CONN_CTE_TX)
 /**
- * @brief Function initializes transmission of Constant Tone Extension.
+ * @brief Function configures transmission of Constant Tone Extension.
  *
  * @param cte_type   Type of the CTE
  * @param cte_length Length of CTE in units of 8 us
- * @param ant_ids_len Length of antenna identifiers array
+ * @param ant_sw_len Length of antenna switch pattern array
```

Comment:
Yeah, it would be better to use same convention as in `direction.h` -> `num_ant_ids`. I have to align it in lll_df and ull_df but I'll do that in separate PR.